.github/workflows/pr-assign.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   auto-request-review:
     runs-on: ubuntu-latest
-    if: ${{ !github.event.pull_request.draft }}
+    if: ${{ !github.event.pull_request.draft && github.event.pull_request.user.type != 'Bot'  }}
```

Comment:
As far as I can tell `pull_request.user.type` isn't strongly typed with a fixed set of values - whomp whomp

Reply:
Yeah I think it's just a string. I'd have to hunt down the API doc I was referencing. Just saw a list of values, but no mention of a type or enum 😕